firmware/ec/test/Makefile

```diff
@@ -4,8 +4,8 @@
 #   [Released under MIT License. Please refer to license.txt for details]
 # ==========================================
 
-UNITY_ROOT = /Users/kashi/work/tools/Unity
-OCWARE_ROOT = ..
+UNITY_ROOT ?= ../../../../../Unity
```

Comment:
I think there should be one less "../" here compared to OCWare repo, since 1 level of directories was removed ("software/OCWare/firmware" is now "firmware/ec")